parsers/iceland.py

```diff
@@ -0,0 +1,93 @@
+import sys
+import requests
+import json
+import csv
+import io
+
+from collections import defaultdict
+from datetime import datetime, timedelta
+from .utils import store_data
+
+# ------------------------------------------------------------------------
+# Globals
+
+URL_CASES_CUM = "https://raw.githubusercontent.com/tryggvigy/CoronaWatchIS/master/data/covid_in_is.cvs"
+URL_HOSPITALIZED_CUM = "https://raw.githubusercontent.com/tryggvigy/CoronaWatchIS/master/data/covid_in_is_hosp.cvs"
+
+LOC = "case-counts/Europe/Western Europe/Iceland"
+cols = ['time', 'cases', 'deaths', 'hospitalized', 'ICU', 'recovered']
+
+# ------------------------------------------------------------------------
+# Functions
+
+def parse_date(s):
+    try:
+      return datetime.strftime(datetime.strptime(s,'%m-%d-%Y'),'%Y-%m-%d')
+    except ValueError:
+        return s
```

Comment:
Why return s upon exception? We only want '%Y-%m-%d' strings

Reply:
Unfortunately the case and hospital data have different date formats

| Case data | Hospital data |
| -| -|
| ![image](https://user-images.githubusercontent.com/2373958/77821086-50923b00-70df-11ea-8bf1-07572c6d0774.png) | ![image](https://user-images.githubusercontent.com/2373958/77821093-5ee05700-70df-11ea-9e35-0d6864c2146c.png) |

Reading the docs I saw that `datetime.strptime` throws a `ValueError` if `s` doesn't match given format (`'%m-%d-%Y'`). I'm assuming that when this happens, `s` is from the **Hospital data** which is already the expected `'%Y-%m-%d'` format.

Hope that explains it! I'm not used to python and maybe this can be handled in a better way.

Reply:
I added code comments in 8f39d7b